english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
@williamfbaur, what you should do is pick any existing page of the story, and add an option, e.g. "make smores" with a link to your new page. Then when the user clicks that option, they will be taken to your page.